capstone/src/main/java/com/google/sps/data/PerspectiveInput.java

```diff
@@ -32,4 +37,8 @@ public String getText() {
   public String getLang() {
     return this.lang;
   }
+
+  public ArrayList<String> getRequestedAttributes() {
+    return this.requestedAttributes;
+	}
```

Comment:
nit: there should be one less tab on the closing bracket

Reply:
Fixed